xml/obs_scm_bridge.xml

```diff
@@ -127,7 +147,7 @@
       by referencing it via the scmsync meta tag.
       </para>
       <para>
-      This has been only implemented for gitea atm, you can find details in the <link xlink:href="https://github.com/adrianschroeter/obs-gitea-bridge">obs-gitea-bridge</link> documentation.
+      This has been only implemented for gitea atm, you can find details in the <link xlink:href="https://github.com/openSUSE/obs-gitea-bridge">obs-gitea-bridge</link> documentation.
```

Comment:
The SCM bridge has so far only been implemented for gitea. You can find details in . . .

Reply:
not the scmsync, but the update notification. changed that now.